lib/features/notes/presentation/pages/note_create_page.dart

```diff
@@ -38,6 +40,8 @@ class _NoteCreatePageState extends State<NoteCreatePage> {
   late double topPadding = 0;
   Timer? _saveTimer;
 
+  final Logger _logger = Logger(); // Initialize the logger
+
```

Comment:
use 
```
  final log = printer("NoteCreatePage");
```

Reply:
i think the printer function doesnt work like that . Its something like this
 var logger = Logger(
  filter: null, // Use the default LogFilter (-> only log in debug mode)
  printer: PrettyPrinter(), // Use the PrettyPrinter to format and print log
  output: null, // Use the default LogOutput (-> send everything to console)
);
Please correct me if i am wrong

Reply:
Hey, `printer` function in `lib/core/logger/logger.dart` is already meant to do same thing, no need to redefine it again

Reply:
I just changes it . Could you please check it?